test/framework/src/main/java/org/opensearch/test/rest/OpenSearchRestTestCase.java

```diff
@@ -900,7 +900,7 @@ private Set<String> runningTasks(Response response) throws IOException {
         return runningTasks;
     }
 
-    protected static void assertOK(Response response) {
+    public static void assertOK(Response response) {
```

Comment:
Why was this changed?

Reply:
To be honest, I suspect it was a suggested fix by the IDE that I missed at somepoint when debugging. Reverted